docs/quick_start.md

```diff
@@ -26,9 +26,21 @@ Paste the following in a macOS Terminal and press enter to install Homebrew.
 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
 ```
 
+Then follow the steps in the terminal screen.
```

Comment:
I would maybe move this in the "Paste the following in a macOs..." line? Make it something like

Paste the following command in a macOS Terminal and follow the steps in the terminal screen

I don't think we need to instruct the user to press enter